package.json

```diff
@@ -38,39 +38,39 @@
     "validate.js": "^0.13.1"
   },
   "devDependencies": {
-    "@commitlint/cli": "^19.5.0",
-    "@commitlint/config-conventional": "^19.5.0",
-    "@digitak/esrun": "^3.2.25",
+    "@commitlint/cli": "^19.6.0",
+    "@commitlint/config-conventional": "^19.6.0",
+    "@digitak/esrun": "^3.2.26",
     "@mui/types": "^7.2.18",
-    "@next/eslint-plugin-next": "^14.1.0",
+    "@next/eslint-plugin-next": "^15.0.3",
     "@types/mdx": "^2.0.11",
-    "@types/node": "^20.10.0",
-    "@types/react": "^18.3.5",
+    "@types/node": "^22.10.1",
```

Comment:
`@types/node` should be matched with the Node version, so I think this should stay at `^20.10.0` (and I believe this is a convention for other type packages, so we might want to keep some of the others the same as well)